kbs/src/api/src/attestation/coco/grpc.rs

```diff
@@ -125,6 +126,27 @@ impl Attest for GrpcClientPool {
 
         Ok(token)
     }
+
+    async fn generate_supplemental_challenge(
+        &self,
+        tee: Tee,
+        tee_parameters: Option<String>,
+    ) -> Result<String> {
+        let req = tonic::Request::new(ChallengeRequest {
+            tee: to_grpc_tee(tee).into(),
+            tee_params: tee_parameters.unwrap(),
```

Comment:
we shouldn't use unwrap/panic in library code. you can turn a None option into a Result using `Option::ok_or()`.

Do we have to use Option for `tee_parameters`? on kbs_types' Request type, the extra_params propery is [not an option](https://github.com/virtee/kbs-types/blob/3f70b78f18e39cfcd3c5225f56b49117c95ee79b/src/lib.rs#L48)

Reply:
Sounds good to me, will fix it.